src/celo/formatters.ts

```diff
@@ -53,8 +53,14 @@ export const formatters = {
     format(args: CeloRpcTransaction): CeloTransaction {
       const transaction = { feeCurrency: args.feeCurrency } as CeloTransaction
 
-      if (args.type === '0x7b') transaction.type = 'cip64'
-      else {
+      if (args.type === '0x7a') {
+        transaction.type = 'cip66'
+        transaction.maxFeeInFeeCurrency = args.maxFeeInFeeCurrency
```

Comment:
not sure if that's important, would `0n` if it's an explicit zero or would you rather have null?

Reply:
if the type is cip66 then this will always be truthy anyway.